experimental/PropertyDDS/packages/property-inspector-table/package.json

```diff
@@ -21,10 +21,11 @@
   ],
   "scripts": {
     "build": "concurrently npm:build:compile npm:lint",
-    "build:compile": "concurrently npm:tsc npm:build:esnext npm:webpack:svgs",
+    "build:compile": "concurrently npm:tsc npm:build:esnext npm:build:webpack",
     "build:esnext": "tsc --project ./tsconfig.esnext.json",
     "build:full": "npm run build",
     "build:full:compile": "npm run build:compile",
+    "build:webpack": "webpack --config webpack.svgs.js && copyfiles -u 2 \"dist/assets/**/*\" \"lib/assets\"",
```

Comment:
I'm not sure this change makes sense. Typically, our "build:webpac" script uses tsloader to compile the typescript and webpack it into a bundle which is not part of the published backage. This case is different in that it uses webpack to process svg files that are part of the build output for this package.

This means that unlike our other ussages of webpack, this one is part of the build that ther packages depend on. While fluid build reads these files and complains if they don't match what it expects, it does not actually run the full build operation before building packages that depend on it: it instead makes assumptions about what dependencies actually exist and does not wait for things it think are not needed, including build:webpack.

This means that all packages that depend on this one MUST explicitly declare that they depend on this build task, or fluid build will have a datarace. If you search package files for "webpack:svgs" you will find examples of this. At the very least these need to be updated.

Personally, I find it very bad that package.json files can't easily have documentation about this kind of thing, and also really bad that packages that use this one need to be special cased, and also bad that you can break this and not break CI. This combines to be a very fragile situation, and great care must be taken dealing with it.

Reply:
Had you and Curtis not pointed it out I would've never known about it. It definitely seems problematic that I could've broken this without CI catching it. As for the script being called "build:webpack,", this is the closest script name that's allowed in the build step that I could find to remove the warning. I've updated the references to "webpack:svgs," let me know if that's sufficient or if we want to go a different route.